src/rules/no_eval.rs

```diff
@@ -84,14 +124,42 @@ impl<'c> Visit for NoEvalVisitor<'c> {
 #[cfg(test)]
 mod tests {
   use super::*;
-  use crate::test_util::*;
 
   #[test]
-  fn no_eval_test() {
-    assert_lint_err::<NoEval>(r#"eval("123");"#, 0);
-    // TODO These tests should pass (#466)
-    // assert_lint_err::<NoEval>(r#"(0, eval)("var a = 0");"#, 0);
-    // assert_lint_err::<NoEval>(r#"var foo = eval;"#, 0);
-    // assert_lint_err::<NoEval>(r#"this.eval("123");"#, 0);
+  fn no_eval_valid() {
+    assert_lint_ok! {
+      NoEval,
+      "foo.eval('bar');",
+    }
+  }
+
+  #[test]
+  fn no_eval_invalid() {
+    assert_lint_err! {
+      NoEval,
+      "eval('123');": [{col: 0, message: MESSAGE, hint: HINT}],
+      "(0, eval)('var a = 0');": [{col: 4, message: MESSAGE, hint: HINT}],
+      "((eval))('var a = 0');": [{col: 2, message: MESSAGE, hint: HINT}],
+      "var foo = eval;": [{col: 4, message: MESSAGE, hint: HINT}],
+
+      // TODO
```

Comment:
Why not add a link to this PR `https://github.com/denoland/deno_lint/pull/490` here for other developers to understand the context?